feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-product-images/product-image-zoom-product-images.component.html

```diff
@@ -8,14 +8,21 @@
 </ng-container>
 
 <ng-container *ngIf="thumbs$ | async as thumbs">
-  <cx-carousel
-    *ngIf="thumbs.length"
-    class="thumbs"
-    [items]="thumbs"
-    itemWidth="120px"
-    [hideIndicators]="false"
-    [template]="thumb"
-  ></cx-carousel>
+  <ng-container *ngIf="getProduct() | async as product">
+    <cx-carousel
+      role="region"
```

Comment:
This is another instance where aria attributes shouldn't really be on the component element tag but rather within it, like the TabParagraphContainer.

Reply:
I see what you mean, but one thing thought is that I don't think we want to make every single `cx-carousel` a region. And the accessibility label is also specific to a product: 'Carousel, Images for {{product}}'